src/main/java/seedu/address/model/patient/Patient.java

```diff
@@ -102,7 +102,6 @@ public boolean equals(Object other) {
 
         Patient otherPatient = (Patient) other;
         return otherPatient.getName().equals(getName())
-                && otherPatient.getTemperature().equals(getTemperature())
                 && otherPatient.getPeriodOfStay().equals(getPeriodOfStay())
```

Comment:
hmm I think the same patient object should still have the same temperature

Reply:
Accidentally deleted.